db/migrate/20211129090243_create_plans.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePlans < ActiveRecord::Migration[6.1]
+  def change
+    create_table :plans do |t|
+      t.string :name
+      t.integer :duration
```

Comment:
what will we store in this for a yearly plan?

Reply:
storing count of days = 365
Now, changed to duration_in_days.